### PR TITLE
Add MinGW-w64 icon

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
image

**Popularity metric:**
https://www.similarweb.com/website/mingw-w64.org/#overview

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
I got this from the official website.

### CODE REVIEW

**Comment by uncenter** [comment]
Please add a preview image!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9973,6 +9973,11 @@
             "hex": "53AC56",
             "source": "https://www.minetest.net"
         },
+        {
+            "title": "minGW",
```
Comment (uncenter) [inline _data/simple-icons.json:9977]:
```suggestion
            "title": "MinGW-w64",
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by MichaelHinrichs** [comment]
> Please add a preview image!

Dang it, I forgot!

File: icons/MinGW-w64.svg
Comment (uncenter) [inline icons/MinGW-w64.svg:1]: The filename should be `icons/mingww64.svg` (see `npm run get-filename`).

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
Please see the linter errors. Same issue with precision as the other one.

File: icons/mingww64.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m-.0003 9.4029 4.187-4.187 4.1869 4.187-4.187 4.187Zm0 10.417 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.1869-4.187 4.187 4.187-4.187 4.187zm0-10.417 4.187-4.187 4.1869 4.187-4.187 4.1868Zm5.2085 5.2084 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2085-5.2084 4.187-4.187 4.1869 4.187-4.187 4.1869zm-5.208 15.626 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.187-4.187 4.1868 4.187-4.1869 4.187z"/></svg>
```
Comment (uncenter) [inline icons/mingww64.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m-.0003 9.4029 4.187-4.187 4.1869 4.187-4.187 4.187Zm0 10.417 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.1869-4.187 4.187 4.187-4.187 4.187zm0-10.417 4.187-4.187 4.1869 4.187-4.187 4.1868Zm5.2085 5.2084 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2085-5.2084 4.187-4.187 4.1869 4.187-4.187 4.1869zm-5.208 15.626 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.187-4.187 4.1868 4.187-4.1869 4.187z"/></svg>
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/mingww64.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m-.0003 9.4029 4.187-4.187 4.1869 4.187-4.187 4.187Zm0 10.417 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.1869-4.187 4.187 4.187-4.187 4.187zm0-10.417 4.187-4.187 4.1869 4.187-4.187 4.1868Zm5.2085 5.2084 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2085-5.2084 4.187-4.187 4.1869 4.187-4.187 4.1869zm-5.208 15.626 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.187-4.187 4.1868 4.187-4.1869 4.187z"/></svg>
```
Comment (adamrusted) [inline icons/mingww64.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MinGW-w64</title><path d="m-.0003 9.4029 4.187-4.187 4.1869 4.187-4.187 4.187Zm0 10.417 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.1869-4.187 4.187 4.187-4.187 4.187zm0-10.417 4.187-4.187 4.1869 4.187-4.187 4.1868Zm5.2085 5.2084 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2085-5.2084 4.187-4.187 4.1869 4.187-4.187 4.1869zm-5.208 15.626 4.187-4.1869 4.1869 4.187-4.187 4.1869zm5.2086-5.2085 4.187-4.187 4.1868 4.187-4.1869 4.187z"/></svg>
```

**Comment by MichaelHinrichs** [comment]
I can't delete the newline!

**Comment by uncenter** [comment]
Are you editing it on the GitHub website? It will always add one for you, so you can't remove it from there.

**Comment by MichaelHinrichs** [comment]
That's better!

**Review by LitoMore — APPROVED** [review]

**Pull request merged**